tests/tck/features/optimizer/PrunePropertiesRule.feature

```diff
@@ -569,28 +569,20 @@ Feature: Prune Properties rule
     And wait 6 seconds
     When executing query:
       """
-      match (v) return properties(v).name limit 2;
+      match (v:player) return properties(v).name AS name order by name limit 2;
       """
     Then the result should be, in any order, with relax comparison:
-      | properties(v).name  |
+      | name                |
```

Comment:
Use regex to fix it.

Reply:
more certainty by sorting results

Reply:
Change origin query will lose the point of test.